CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Add a mixin to allow Location types
```

Comment:
IMO publicly we should refer to these as "constants" rather than "mixins" - it's clearer for people who aren't familiar with the mechanism they're implemented with.

Reply:
Good point! Will fix it on master so to not have to wait for another build :D